### DISCUSSION TITLE
Which backend do you use with `youtube-dl-wpf`?

### DISCUSSION THREAD

**Discussion by database64128** [database64128/youtube-dl-wpf#127] [Polls]
`youtube-dl-wpf` currently supports both `youtube-dl` and `yt-dlp`. The rise of `yt-dlp` was largely due to `youtube-dl`'s lack of maintenance. `youtube-dl` hadn't really been active for the better half of 2021. At the beginning of 2022, `youtube-dl` was taken over by some new maintainers, and the primary goal of the project became "backward compatibility with Python 2.6". The future of `youtube-dl`, as outlined by the new maintainers in ytdl-org/youtube-dl#30568, isn't really something I agree with.

I'm considering dropping `youtube-dl` support. Dropping `youtube-dl` makes it easier to have better integrations with `yt-dlp`. There have already been features that are currently difficult to implement due to the differences between `youtube-dl` and `yt-dlp`. GUI for external downloaders (#83) is one example.

Do you still use `youtube-dl-wpf` with `youtube-dl`? Will this affect you? Feel free to participate in the poll and leave comments.

**Comment by EDM115** [comment]
yt-dlp is for sure the most used nowadays
